api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -589,6 +589,7 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
             "CREATE TABLE ml.delegations (
                     delegation_id TEXT NOT NULL,
                     block_height bigint NOT NULL,
+                    creation_block bigint NOT NULL,
```

Comment:
As discussed, please call it creation_block_height, and same in all relevant places.